src/main/java/de/srendi/advancedperipherals/common/blocks/blockentities/RedstoneIntegratorEntity.java

```diff
@@ -82,6 +82,11 @@ private void updateRedstoneOutputs() {
      * @param power     The redstone power from 0 to 15
      */
     public void setOutput(Direction direction, int power) {
+        if (power > 15) {
+            power = 15;
+        } else if (power < 0) {
+            power = 0;
+        }
```

Comment:
What do you think about

```suggestion
        power = Math.min(Math.max(power, 0), 15);
```